jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/ActionPrefixesR.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.fuseki.servlets;
+
+import com.google.gson.JsonArray;
+import org.apache.jena.atlas.logging.FmtLog;
+import org.apache.jena.riot.WebContent;
+import org.apache.jena.riot.web.HttpNames;
+
+import org.apache.jena.fuseki.servlets.prefixes.ActionPrefixesBase;
+import org.apache.jena.fuseki.servlets.prefixes.PrefixUtils;
+import org.apache.jena.fuseki.servlets.prefixes.JsonObject;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class ActionPrefixesR extends ActionPrefixesBase {
+
+    private static final String NO_PREFIX_NS = "";
+
+    public ActionPrefixesR() {}
+
+    @Override
+    protected void doOptions(HttpAction action) {
+        ActionLib.setCommonHeadersForOptions(action);
+        action.setResponseHeader(HttpNames.hAllow, "GET,OPTIONS");
+        ServletOps.success(action);
+    }
+
+    public void validateGet(HttpAction action) {
+        validate(action);
+        // check if the combination of parameters is legal
+        String prefix = action.getRequestParameter("prefix");
```

Comment:
Can we have all the parameter and JSON field names defined as constants somewhere please?